internal/vcs/git/blob.go

```diff
@@ -38,7 +38,53 @@ func ReadFile(ctx context.Context, repo gitserver.Repo, commit api.CommitID, nam
 	return b, nil
 }
 
+// GetFileReader returns an io.ReadCloser reading from the named file at commit.
+// The caller should always close the reader after use
+func GetFileReader(ctx context.Context, repo gitserver.Repo, commit api.CommitID, name string) (io.ReadCloser, error) {
```

Comment:
It seems a new reader is created by calling this function, WDYT about this?

```suggestion
func NewFileReader(ctx context.Context, repo gitserver.Repo, commit api.CommitID, name string) (io.ReadCloser, error) {
```